.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,58 @@
+name: Feature Request
+description: Request a new feature
+labels: ["Type: enhancement"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to share your feedback on ways Apache Arrow can be improved!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the feature requested.
+    validations:
+      required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component
+      description: Which component does the feature most apply to?
```

Comment:
Hmm, do we have to repeat this one exactly?

Reply:
Removed in https://github.com/apache/arrow/pull/14688.